typescript/simple-mercari-web/src/components/ItemList/ItemList.tsx

```diff
@@ -46,20 +46,19 @@ export const ItemList: React.FC<Prop> = (props) => {
   }, [reload]);
 
   return (
-    <div>
-      {items.map((item) => {
+    <div className="grid-container">
+      {items.map((item, index) => {
         return (
-          <div key={item.id} className='ItemList'>
-            {/* TODO: Task 1: Replace the placeholder image with the item image */}
-            <img src={placeholderImage} />
+          <div key={item.id} className="item">
+            <img src={`${server}/image/${item.image}`} height="200" alt={item.name} />
```

Comment:
This height can be removed (maybe forgot to remove?), because you already set height of `.item img` by CSS.
```suggestion
            <img src={`${server}/image/${item.image}`} alt={item.name} />
```